### ISSUE TITLE
What is the model prediction format in training and how can extract bboxes and their confidences from it?

### ISSUE THREAD

**Issue by jafariparizi** [ultralytics/yolov5#9513]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello. Model prediction in `train.py` occurs in: https://github.com/ultralytics/yolov5/blob/095f601d9d32ea0f0afd47554c068659939ecf4e/train.py#L306-L307 Then `pred` is passed to `loss.py` in:
https://github.com/ultralytics/yolov5/blob/095f601d9d32ea0f0afd47554c068659939ecf4e/train.py#L308 In Loss, prediction and target boxes are passed to:https://github.com/ultralytics/yolov5/blob/095f601d9d32ea0f0afd47554c068659939ecf4e/utils/loss.py#L141 For iou. The tensor of pbox is like: 

```
tensor([[ 3.9697e-01,  4.6191e-01,  5.7057e+00,  4.0823e+00],
        [ 4.0527e-01,  5.1562e-01,  5.1097e+00,  4.6274e+00],
        [ 4.7949e-01,  4.8730e-01,  5.0741e+00,  3.9688e+00],
	.
	.
	.
        [ 6.5039e-01,  4.1309e-01,  1.3253e+01,  6.5910e+00],
        [ 3.2471e-01,  1.9043e-01,  1.6242e+01,  5.3213e+00]], device='cuda:0',
       requires_grad=True)
```
I need to extract boxes and their confidences from this tensor. what is the format of it? thanks

### Additional

The `tbox` tensor is the same as below:

```
tensor([[ 0.9146,  0.5309,  5.0402,  3.6122],
        [ 0.5829,  0.2181,  2.4817,  1.4416],
        [ 0.6427,  0.5008,  3.3440,  3.0760],
	.
	.
	.
        [ 0.9146, -0.4691,  5.0402,  3.6122],
        [ 0.6427, -0.4992,  3.3440,  3.0760],
        [ 0.9522, -0.0292,  6.5766,  6.1992]], device='cuda:0')
```

**Comment by glenn-jocher** [comment]
@jafariparizi hello there! The tensor you shared represents bounding box predictions, where each row corresponds to a single prediction in (x_center, y_center, width, height) format, along with confidences.

To extract the bounding boxes and their confidences from the tensor, you can use the following indices and slicing to separate the values:

- Class confidences: `tensor[:, 4:]`
- Bounding box coordinates: `tensor[:, :4]`

You can then use these extracted values for post-processing, such as non-maximum suppression or visualization.

It's great to see you exploring the inner workings of YOLOv5. If you need further assistance, feel free to refer to the YOLOv5 [documentation](https://docs.ultralytics.com/yolov5/) or reach out with more questions. Keep up the great work!
